contracts/seed/Seed.sol

```diff
@@ -113,6 +113,7 @@ contract Seed {
       * @param _fundingToken          The address of the token being exchanged for seed token.
       * @param _successMinimumAndCap  Array of the minimum distribution threshold and
                                       the highest possible amount to be raised.
+      * @param _price                 Seed to funding token exchange rate.
```

Comment:
I had to read this about four times to understand it.  Just saying'.

Reply:
Can you please reword this like:  "The price in [what unit] of [X] when exchanged for [Y]"